docs/_posts/examples/3400-01-31-data-join.html

```diff
@@ -0,0 +1,70 @@
+--- 
+layout: example 
+category: example 
+title: Join client data to vector tiles 
+description: Use data-driven styles to join client data to vector tiles
+tags: 
+  - sources 
+---
+
+<div id='map'></div>
+<script>
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/light-v9',
+    center: [-99.9, 41.5],
+    zoom: 1
+});
+
+// Generate client data to join to vector tiles
+// Key is the property to join, value is the measure to style on
+// EX: {"01" : 27, "02" : 89}
```

Comment:
@ryanbaumann I agree with @lucaswoj here that a real dataset would help add context to this example. Population for each state might be a good dataset: https://en.wikipedia.org/wiki/2010_United_States_Census#State_rankings . Can you think of a way to drop those numbers in without making the example look too cluttered?